### ISSUE TITLE
KeyVault Certificate Storage in Secret Section is Deprecated

### ISSUE THREAD

**Issue by mikeruhl** [RamjotSingh/PreConfiguratorForTraefikOnServiceFabric#8]
Hey @RamjotSingh , excellent tool.  I noticed that it is attempting to get the certificate from the secret store in KeyVault, [this is deprecated functionality](https://github.com/MicrosoftDocs/azure-docs/issues/7977).  The proper store now is the certificate store.  I have updated the code in a fork if you'd like to look at it.  

I also updated the post-build copy command to better target the output directory.
Here's the commit:
[updated keyvault access by retrieving certificate instead of secret](https://github.com/mikeruhl/PreConfiguratorForTraefikOnServiceFabric/commit/f0e0ec4820125b8204cd5815abb063e1f8c56e4f)

**Comment by RamjotSingh** [comment]
I will have to dig into the code to see if this has been deprecated. The issue links to another part. That deprecation is the ability to import PFX files as secrets directly. That part has been deprecated by KeyVault. The code that exists right now, already depends that you must have imported the certificate using the certificate tab. 

https://github.com/aspnet/Extensions/blob/master/src/Configuration/Config.AzureKeyVault/src/AzureKeyVaultConfigurationProvider.cs also seems to use the same way being used right now. The portal experience and SDK are 2 different things. Certificates uploaded using certificate tab are stored as managed secrets (keyvaulturl/secrets response can give you more into).

**Comment by mikeruhl** [comment]
Hey, you're right, and the last thing I want to do is make a breaking change.  I updated my code to allow for either secrets or certificates while remaining backwards-compatible.  I did this by introducing 2 new sources: `KeyVaultCertificate` and `KeyVaultSecret`.  Thanks for the feedback!
[5320950](https://github.com/mikeruhl/PreConfiguratorForTraefikOnServiceFabric/commit/5320950818e725667564520802dce8eb084bc1a7)
